docs/doc/15-sql-functions/60-conversion-functions/index.md

```diff
@@ -4,13 +4,33 @@ title: 'Conversion Functions'
 
 import FunctionDescription from '@site/src/components/FunctionDescription';
 
-<FunctionDescription description="Introduced or updated: v1.2.150"/>
+<FunctionDescription description="Introduced or updated: v1.2.187"/>
 
-Below is a list of functions that allow you to convert an expression from one data type to another.
+This page lists functions that allow you to convert an expression from one data type to another.
 
-:::tip
-Databend also offers a variety of functions for converting expressions into different date and time formats. Check out the **SQL Functions** > **Date & Time Functions** section to explore them.
-:::
+- When converting from floating-point, decimal numbers, or strings to integers or decimal numbers with fractional parts, Databend rounds the values to the nearest integer. This is determined by the setting `numeric_cast_option` (defaults to 'rounding') which controls the behavior of numeric casting operations. When `numeric_cast_option` is explicitly set to 'truncating', Databend will truncate the decimal part, discarding any fractional values.
```

Comment:
When casting String to Int, if the string is a floating point number, we will report an error.